webapp/sources/rudder/rudder-core/src/main/resources/ldap/rudder.schema

```diff
@@ -356,6 +356,13 @@ attributetype ( RudderAttributes:307
   SUBSTR caseIgnoreSubstringsMatch
   SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 
+attributetype ( RudderAttributes:308
+  NAME 'inheritMode'
+  DESC 'inheritance mode like merge, overrid, etc for properties and parameters object/array/string'
```

Comment:
```suggestion
  DESC 'inheritance mode like merge, override, etc for properties and parameters object/array/string'
```

You can accept suggestion by prefixingby fixup! when accepting

Reply:
It will need a rebase in all case

Reply:
Non it will create a new commit and it will be rebased if you add fixup! to the commit message

Reply:
No, I mean that I will have to do a manual rebase because there is already existing commit that are not fixup